src/DssSpell.t.base.sol

```diff
@@ -1878,6 +1908,11 @@ contract DssSpellTestBase is Config, DssTest {
 
         //////////////////////////////////////////
 
+        // If neither the version or the content have changed, there is nothing to test
+        if (cacheAfter.versionHash == cacheBefore.versionHash && cacheAfter.contentHash == cacheBefore.contentHash) {
+            vm.skip(true);
+        }
```

Comment:
I'm actually not sure if I like this approach of self-skipping tests or not. Maybe all other periodically used tests should do the same (not implying you need to do it in the same PR)? E.g. instead of manually setting the modifier, DAI vesting test could skip itself as soon as there are no vesting changes, but run and fail (with previous values) as soon as there is some change

Reply:
> I'm actually not sure if I like this approach of self-skipping tests or not.

I believe it's better to explicitly skip than running a test with no assertions and having it pass. At least, the observers can know for sure nothing is being tested.

> Maybe all other periodically used tests should do the same (not implying you need to do it in the same PR)?

We'd need to investigate how to do that. Each test will be different.
- For vests, we could check the number of streams before and after the spell is the same. If it is not, then it fails.
- For payments from the surplus buffer, we could get the amount there before the spell and the one after and check the difference regarding some threshold to determine if a payment was done.
  - We need a threshold in case the spell itself changes SB (i.e.: `Jug.drip` are made within the spell)